src/components/landing/CommunityListingHeader.js

```diff
@@ -0,0 +1,98 @@
+import CustomizableAvatar from "../common/customavatar/CustomizableAvatar";
+import Link from "next/link";
+import Image from "next/image";
+
+const CommunityHeader = ({ listing }) => {
+  const photo = "/community dashoard  img 1.jpg";
+  const Manager_Avatar = "/avarar 1.png";
+  const AVATAR_WIDTH = 50;
+  const AVATAR_HEIGHT = 50;
+  const SmallAvatar_WIDTH = 20;
+  const SmallAvatar_HEIGHT = 20;
+  const LOGO_WIDTH = 20;
+  const LOGO_HEIGHT = 20;
+  return (
+    <header className="">
+      <div className=" bg-primary-lite p-5 md:flex lg:flex items-center justify-between px-10">
+        <article className="flex items-center pb-4">
+          <CustomizableAvatar
+            src={photo}
+            width={AVATAR_WIDTH}
+            height={AVATAR_HEIGHT}
+            avatarImage={"rounded-full"}
+            avatarCustomClass={"mx-2"}
+          />
+          <h1 className="text-grey-dark text-xl font-medium capitalize">
+            {listing?.community_name}
+          </h1>
+        </article>
+
+        <div className="flex items-center px-4">
+          <div className="flex items-center bg-white rounded-lg py-2 px-4 text-justify">
+            <CustomizableAvatar
+              src={Manager_Avatar}
+              width={SmallAvatar_WIDTH}
+              height={SmallAvatar_HEIGHT}
+              className={"rounded-full"}
+            />
+            <p className="text-primary text-sm text-center pl-1 capitalize">
+              {listing?.community_manager}
```

Comment:
@annonymousauthority Please advise us here: I generally do not like the use of `.?` in code (unless absolutely unavoidable). I consider it a poor design pattern. is there a way to check ONCE for the existence of `listing` and then render our components accordingly? 🙏🏾